src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/content-browser.component.ts

```diff
@@ -187,6 +187,20 @@ export class ContentBrowserComponent extends BaseBrowserComponent<Content> imple
     return containClasses.some(iClass => elementClassName.includes(iClass));
   }
 
+  getFilter(): FilterBase | undefined {
+    if (this.baseFilter && this.filter) {
+      return new AndFilter({ filters: [this.baseFilter, this.filter] });
+    }
+
+    if (this.baseFilter) {
```

Comment:
Does the baseFilter have a bigger order of precedence than the filter?
Or should we check the filter first and then the baseFilter?